docs/usage/shoot_status.md

```diff
@@ -50,30 +50,45 @@ For example, if a webhook has rules for `CREATE/UPDATE` Pods or Nodes and `failu
 Even if the `failurePolicy` is set to `Ignore`, high timeouts (`>15s`) can lead to blocking requests of control plane components.
 That's because most control-plane API calls are made with a client-side timeout of `30s`, so if a webhook has `timeoutSeconds=30`
 the overall request might still fail as there is overhead in communication with the API server and potential other webhooks.
-Generally, it's [best pratice](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#timeouts) to specify low timeouts in WebhookConfigs.
-Also, it's [best practice](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#avoiding-operating-on-the-kube-system-namespace)
-to exclude the `kube-system` namespace from webhooks to avoid blocking critical operations on system components of the cluster.
-Shoot owners can do so by adding a `namespaceSelector` similar to this one to their webhook configurations:
+Generally, it's [best practice](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#timeouts) to specify low timeouts in WebhookConfigs.
+
+As an effort to correct this common problem, the webhook remediator has been created. This is enabled by setting `.controllers.shootCare.webhookRemediatorEnabled=true` in the Gardenlet configuration. This feature simply checks whether a customer's webhook matches a set of rules, described [here](https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/matchers/matcher.go). If at least one of the rule matches, it will change the constraints (`HibernationPossible` and `MaintenancePreconditionsSatisfied`) statuses. In such cases, the `failurePolicy` will be set from `Fail` to `Ignore`. Gardenlet will also add an annotation to make it visible to end-users that their webhook configurations were mutated and should be fixed by them in the first place.
```

Comment:
```suggestion
As an effort to correct this common problem, the webhook remediator has been created. This is enabled by setting `.controllers.shootCare.webhookRemediatorEnabled=true` in the `gardenlet`'s configuration. This feature simply checks whether webhook configurations in shoot clusters match a set of rules described [here](https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/matchers/matcher.go). If at least one of the rules matches, it will change set `status=False` for the `.status.constraints` of type `HibernationPossible` and `MaintenancePreconditionsSatisfied` in the `Shoot` resource. In addition, the `failurePolicy` in the affected webhook configurations will be set from `Fail` to `Ignore`. Gardenlet will also add an annotation to make it visible to end-users that their webhook configurations were mutated and should be fixed/adapted according to the rules and best practices.
```